### DISCUSSION TITLE
QuickAdd: (ERROR) failed to run user script movies. Error: Request failed, status 401

### DISCUSSION THREAD

**Discussion by xXxWoogiexXx** [chhoumann/quickadd#652] [Q&A] [answered]
Hi I'm getting an error as said in the title when running the command I made through quickadd to add movies using Omdb. I've been looking through multiple tutorials and redownloading the movies script in multiple different ways with no luck. I have separated quickadd and templator scripts with folders as I have seen a bug/clash when they're in the same folder, which didn't work either. Any suggestions? Thanks

**Comment by pcuellar** [comment, marked as answer]
401 is related to the API key. See https://github.com/chhoumann/quickadd/discussions/553

**Comment by chhoumann** [comment]
Coming back to this one late - sorry about that. pcuellar is right: a 401 means OMDb rejected the API key, so it is not something on QuickAdd's side. Re-check the key you pasted into the movies script and make sure you clicked the activation link in the email OMDb sends - #553 walks through the same problem. Marking that comment as the answer.
